projects/storefrontapp-e2e-cypress/cypress/integration/accessibility/aux-key.e2e-spec.ts

```diff
@@ -0,0 +1,270 @@
+import { testProductUrl } from '../../helpers/accessibility/tabbing-order';
+
+context('Auxiliary Keys', () => {
+  describe('Category Navigation', () => {
+    before(() => {
+      cy.server();
+      cy.route(
+        `${Cypress.env('API_URL')}/rest/v2/electronics-spa/cms/components*`
+      ).as('getComponents');
+      cy.visit('/');
+      cy.wait('@getComponents');
+    });
+
+    it('should open menu with space key', () => {
+      cy.get('cx-page-slot[ng-reflect-position="NavigationBar"]').within(() => {
+        cy.get('cx-navigation-ui')
+          .find('nav')
+          .should('have.length', 30);
+        cy.get('cx-navigation-ui nav h5')
+          .contains('Brands')
+          .should('be.visible');
+        cy.wait(1000); // TODO: Wait stabilizes test, change after cx-navigation-ui refactor
+        cy.get('cx-navigation-ui nav span')
+          .first()
+          .focus();
+        cy.focused().trigger('keydown', {
+          key: ' ',
+          code: 'Space',
+          force: true,
+        });
+        cy.get('cx-navigation-ui nav div[class="wrapper"]').should(
+          'be.visible'
+        );
+      });
+    });
+
+    it('should tab through menu items', () => {
+      cy.pressTab();
+      cy.focused().should('contain.text', 'Shop all Brands');
+      cy.pressTab();
+      cy.focused().should('contain.text', 'Canon');
+      cy.pressTab();
+      cy.focused().should('contain.text', 'Sony');
+      cy.pressTab();
+      cy.focused().should('contain.text', 'Kodak');
+      cy.pressTab();
+      cy.focused().should('contain.text', 'Samsung');
+      cy.pressTab();
+      cy.focused().should('contain.text', 'Toshiba');
+      cy.pressTab();
+      cy.focused().should('contain.text', 'Fujifilm');
+      cy.pressTab();
+      cy.focused().should('contain.text', 'Kingston');
+      cy.pressTab();
+      cy.focused().should('contain.text', 'Icidu');
+      cy.pressTab();
+      cy.focused().should('contain.text', 'TDK');
+      cy.pressTab();
+      cy.focused().should('contain.text', 'Sweex');
+    });
+
+    it('should close menu with space key', () => {
+      cy.focused().type(' ');
+      cy.get('cx-navigation-ui nav span')
```

Comment:
maybe instead of complicating this, we should go with the opposite of what we did before:
```
cy.get('cx-navigation-ui nav div[class="wrapper"]').should(
    'be.visible' // change this to should not be visible
);
```